app/controllers/organizations/staff/invitations_controller.rb

```diff
@@ -74,4 +82,12 @@ def after_accept_path_for(_resource)
       root_path
     end
   end
+
+  # TODO: We should consider removing this duplicated logic, so we don't have to do this.
```

Comment:
I'm not sure we need this info in both User and Person, especially the names. Think we can remove these from User. 

Removing duplicated email may be a bit more complicated since it's used for auth purposes, so maybe that can remain.

Reply:
Yeah I agree...storing the same data in two places is not good practice. Email is the only one that we might want to do this for until we think of a good solution.